distr_tracing/distr_tracing_rn/distr-tracing-rn-3-0.adoc

```diff
@@ -0,0 +1,105 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="distributed-tracing-rn-3-0"]
+= Release notes for {DTProductName} 3.0
+:context: distributed-tracing-rn-3-0
+
+toc::[]
+
+include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
+
+[id="component-versions_distributed-tracing-rn-3-0"]
+== Component versions in the {DTProductName} 3.0
+
+[options="header"]
+|===
+|Operator |Component |Version
+|{JaegerName}
+|Jaeger
+|1.51.0
+
+|{OTELName}
+|OpenTelemetry
+|0.89.0
+
+|{TempoName}
+|Tempo
+|2.3.0
+|===
+
+// Jaeger section
+[id="jaeger-release-notes_distributed-tracing-rn-3-0"]
+== {JaegerName}
+
+[id="new-features-and-enhancements_jaeger-release-notes_distributed-tracing-rn-3-0"]
+=== New features and enhancements
+
+* Support cluster-wide proxy environments.
+* ARM architecture is now supported.
+
+[id="bug-fixes_jaeger-release-notes_distributed-tracing-rn-3-0"]
+=== Bug fixes
+
+* Fixed support for disconnected environments via `oc adm catalog mirror`. (link:https://issues.redhat.com/browse/TRACING-3546[TRACING-3546])
+
+[id="known-issues_jaeger-release-notes_distributed-tracing-rn-3-0"]
+=== Known issues
+
+* Apache Spark is not supported.
+
+ifndef::openshift-rosa[]
+
+* The streaming deployment via AMQ/Kafka is unsupported on IBM Z and IBM Power Systems.
+endif::openshift-rosa[]
+
+// Tempo section
+[id="tempo-release-notes_distributed-tracing-rn-3-0"]
+== {TempoName}
+
+[id="new-features-and-enhancements_tempo-release-notes_distributed-tracing-rn-3-0"]
+=== New features and enhancements
+
+* Support span request count, duration and error count (RED) metrics. The metrics can be visualized in the Jaeger console deployed as part of Tempo or in the OpenShift console in the observe menu.
+* ARM architecture is now supported.
+
+[id="bug-fixes_tempo-release-notes_distributed-tracing-rn-3-0"]
+=== Bug fixes
+
+* The custom TLS CA option is now fully supported for connecting to object storage. (link:https://issues.redhat.com/browse/TRACING-3462[TRACING-3462])
```

Comment:
@pavolloffay, I had a look at the Jira, and it's not clear to me whether it will be resolved for 3.0.

The wording here reads "is now fully supported", which sounds like a known issue.

Is this a known issue or a bug fix?

And if this is a bug fix indeed, then what do you prefer to replace the phrase "is now fully supported" with?

Reply:
It's a bug fix for "The custom TLS CA option for connecting to object store.". Feel free to phrase it as you prefer.